groups/live/profiles/heritage-live-eu-west-2/vars

```diff
@@ -0,0 +1,50 @@
+# Account details
+aws_profile = "heritage-live-eu-west-2"
+aws_region  = "eu-west-2"
+aws_account = "heritage-live"
+
+# Account shorthand
+account = "hlive"
+region  = "euw2"
+
+# Application details
+environment = "live"
+
+# Jfil Server
+jfil_application       = "Jfil Server"
+jfil_ec2_name          = "jfil-app"
+jfil_ec2_instance_type = "t2.xlarge"
+jfil_ami               = "ami-027676865ead3918d" # rhel6-base-0.1.25
```

Comment:
Should be a data lookup?

Reply:
all of those?

if it's the AMI ID - this is a standalone instance so fixing the ami id instead of a lookup would prevent the risk of TF drift when new AMI's come along.